_blog/2018/traffic-mirroring.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Traffic mirroring with Istio for testing in production"
+overview: An introduction to safer, lower-risk deployments and release to production
+publish_date: February 8, 2018
+subtitle: Routing rules for HTTP traffic
+attribution: Christian Posta
+
+order: 91
+
+layout: blog
+type: markdown
+redirect_from: "/blog/traffic-mirroring.html"
+---
+{% include home.html %}
+
+Trying to enumerate all of the possible combinations of test cases for testing services in non-production/test environments can be daunting. In some cases, you'll find that all of the effort that goes into cataloging these use cases doesn't match up to real production use cases. Ideally, we could use live production use cases and traffic to help illuminate all of the areas of our code that we might miss in more contrived testing environments. 
```

Comment:
all of the -> all the

Reply:
Can we find a better way to say "all of the areas of our code". FOr example, "illuminate all of the feature areas of the service under test"